### PR TITLE
Port API auth config forward from #2724

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** Summary of changes

Addresses [CUMULUS-2853](https://bugs.earthdata.nasa.gov/browse/CUMULUS-2853)

## Changes
  - Move OAUTH_PROVIDER to lambda env variables to address regression in 9.9.2/CUMULUS-2275
  - Add logging output to api app router

## PR Checklist

- [x] Update CHANGELOG
- [ ] Unit tests
- [x] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -40,6 +40,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Fixed
 
+- **CUMULUS-2853**
+  - Move OAUTH_PROVIDER to lambda env variables to address regression in 9.9.2/CUMULUS-2275
```
Comment (markdboyd) [inline CHANGELOG.md:44]: I don't think CUMULUS-2275 is the right ticket number? Should it be 2853?

Comment (Jkovarik) [reply CHANGELOG.md:44]: @markdboyd  the regression was from the changes introduced in CUMULUS-2275.    The ticket in the header *is* 2853.

Comment (Jkovarik) [reply CHANGELOG.md:44]: And..... possibly 2275 as well on this baseline.  I'll check the ticket ID on the main branch.

Comment (markdboyd) [reply CHANGELOG.md:44]: Are you sure? Ticket 2275 seems to be related to CMA 1.3.0 changes?

https://bugs.earthdata.nasa.gov/browse/CUMULUS-2275

Comment (Jkovarik) [reply CHANGELOG.md]: 2781 is the correct ticket 😓.    Updated: c6d5d35ff

**Review by markdboyd — APPROVED** [review]

**Pull request merged**